.github/workflows/tsan.yml

```diff
@@ -0,0 +1,59 @@
+name: tsan
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [20.04, 22.04]
+        platform: [linux/amd64, linux/arm64]
+        target: [clang-shared-regression-tsan]
+    steps:
+    - name: Install qemu-user-static
+      run: |
+        if [[ "${{ matrix.platform }}" == "linux/arm64" ]]; then
+          sudo apt-get update && sudo apt-get install -y qemu-user-static
+        fi
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Build
+      run: |
+        docker buildx build --platform ${{ matrix.platform }} -t systemc_test --build-arg UBUNTU_VERSION=${{ matrix.version }} -f docker/ubuntu.dockerfile .
+        docker run -e SYSTEMC_CI_TARGET=${{ matrix.target }} systemc_test
+  almalinux:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [8, 9]
+        platform: [linux/amd64, linux/arm64]
+        target: [clang-shared-regression-tsan]
+    steps:
+    - name: Install qemu-user-static
+      run: |
+        if [[ "${{ matrix.platform }}" == "linux/arm64" ]]; then
+          sudo apt-get update && sudo apt-get install -y qemu-user-static
+        fi
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Build
+      run: |
+        docker buildx build --platform ${{ matrix.platform }} -t systemc_test --build-arg ALMA_VERSION=${{ matrix.version }} -f docker/alma.dockerfile .
```

Comment:
`alma.dockerfile` -> `almalinux.dockerfile`?

Reply:
The file is called `alma.dockerfile`: https://github.com/accellera-official/systemc/blob/main/docker/alma.dockerfile